Yoast/yoast-components
# Implement help text in react snippet editor

## Summary

This PR can be summarized in the following changelog entry:

* Adds a button to the react snippet editor that toggles the help text

## Relevant technical choices:

* Uses a wrapper that contains the button and the div
* The HelpText component is included from the `feature/content-analysis-in-sidebar` branch and has already been code reviewed

## Test instructions

This PR can be tested by following these steps:

* Use define( 'YOAST_FEATURE_SNIPPET_PREVIEW', true ); in wordpress-seo to see the react snippet editor (for example on line 46 in class-metabox-formatter.php)
* There should be a question mark button in the react snippet preview that should toggle help text above the snippet title
* The appearance of the button and the text should be the same as in the current snippet preview (including focus and hover styles)
* Running `yarn test` should pass all tests 

Fixes #481
